pkg/resource/factory.go

```diff
@@ -93,3 +83,31 @@ func (rf *Factory) SliceFromBytes(in []byte) ([]*Resource, error) {
 	}
 	return result, nil
 }
+
+func (rf *Factory) Set(fs fs.FileSystem, ldr ifc.Loader, wd string) {
+	rf.kf.Set(fs, ldr, wd)
+}
+
+func (rf *Factory) MakeConfigMap(args *types.ConfigMapArgs) (*Resource, error) {
```

Comment:
The method name is self explanatory here, but the linter should demand comment on these public methods.